build.gradle

```diff
@@ -39,6 +39,9 @@ dependencies {
     implementation 'io.micrometer:micrometer-registry-prometheus'
     implementation("io.github.microutils:kotlin-logging-jvm:$ktLoggingVersion")
 
+    implementation("org.springframework.boot:spring-boot-starter-webflux")
```

Comment:
essa implementação é necessária?
Visto que o coroutines é do kotlin

Reply:
> essa implementação é necessária? Visto que o coroutines é do kotlin

Acabei de testar aqui sem a implementação do web-flux e a API rodou normalmente, com os endpoints funcionando, vou remover essa implementação desnecessária

Reply:
commit: [improving logs, change Dispatcher strategy, remove webFlux dependency](https://github.com/the-coding-force/exploring-kotlin-coroutines__backend__spring-kotlin__poc/pull/13/commits/bcb1229dfa29acc3455c10a16adc4b18b4edd81d)